lib/bindings/makefile

```diff
@@ -58,7 +58,7 @@ x86_64-linux-android: $(SOURCES) ndk-home
 bindings-android: android
 	cp -r ffi/kotlin/jniLibs langs/android/lib/src/main
 	cp -r ffi/kotlin/breez_sdk_liquid langs/android/lib/src/main/kotlin/
-	cd android && ./gradlew assemble
+	cd langs/android && ./gradlew assemble
```

Comment:
It seems that the CI doesn't use this script that's why it wasn't detected: https://github.com/breez/breez-sdk-liquid/blob/main/.github/workflows/publish-android.yml#L55